packages/react-mutation-mapper/src/component/column/Annotation.tsx

```diff
@@ -219,6 +234,15 @@ export function getAnnotationData(
                     ? is3dHotspot(mutation, hotspotData.result)
                     : false,
             hotspotStatus: hotspotData ? hotspotData.status : 'pending',
+            vue:
+                indexedVariantAnnotations &&
```

Comment:
this will work but it doesn't make a lot of sense.  you should first check for indexedVariantAnnotations?.isComplete

this will cover 1. case where indexedVariantAnnotations doesn't exist and 2. where isComplete is false.  at that point, result will be populated